app/models/delete_queue_job.rb

```diff
@@ -22,6 +22,9 @@ def perform
       file_type = object.file_type
       study = object.study
 
+      # delete embedded expression_file_info to prevent validation errors later, if present
+      object.expression_file_info.delete if object.expression_file_info.present?
+
```

Comment:
I don't understand why this is necessary.  Is this leftover from troubleshooting?

Reply:
Ah yeah, that's a leftover now.  Since we're not calling `save(validate: false) anymore, this isn't needed.  I was testing to see if preemptively clearing out the `expression_file_info` would help with the update so we could use `update!` instead of `update_attributes` and `save(validate: false)`.